test/e2e/storage/testsuites/provisioning.go

```diff
@@ -178,23 +185,9 @@ func (p *provisioningTestSuite) defineTests(driver TestDriver, pattern testpatte
 		if len(nodes.Items) <= 1 {
 			framework.Skipf("need more than one node - skipping")
 		}
-		l.testCase.PvCheck = func(claim *v1.PersistentVolumeClaim, volume *v1.PersistentVolume) {
-			PVMultiNodeCheck(l.cs, claim, volume, framework.NodeSelection{Name: l.config.ClientNodeName})
-		}
-		l.testCase.TestDynamicProvisioning()
-	})
-
-	It("should create and delete block persistent volumes", func() {
```

Comment:
I removed this test case because I think it is similar to [this one](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/storage/testsuites/volumemode.go#L252)

cc @mkimuram